pkg/testfixtures/storage/mysql.go

```diff
@@ -41,11 +42,29 @@ func (m *mySQLTestContainer) RunMySQLTestContainer(t testing.TB) DatastoreTestCo
 	dockerClient, err := client.NewClientWithOpts(client.FromEnv)
 	require.NoError(t, err)
 
-	reader, err := dockerClient.ImagePull(context.Background(), mySQLImage, types.ImagePullOptions{})
+	allImages, err := dockerClient.ImageList(context.Background(), types.ImageListOptions{
```

Comment:
should these be in a common method?

Reply:
yes! i think it's possible to extract all the common code for both storages.

can I do that in a separate PR?